lib/matplotlib/colors.py

```diff
@@ -613,16 +613,20 @@ def set_bad(self, color='k', alpha=None):
             self._set_extremes()
 
     def set_under(self, color='k', alpha=None):
-        """Set color to be used for low out-of-range values.
-           Requires norm.clip = False
+        """
+        Set color to be used for low out-of-range values.
```

Comment:
"Set the color for low out-of-range values when ``norm.clip = False``." would fit in one line.